lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,40 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * @param {Element} node
+ * @returns {string}
+ */
+/* istanbul ignore next */
+function getNodeTitle(node) {
+  if (node.tagName === 'HTML' || node.tagName === 'BODY') {
+    // too broad, contains all page content
+    return null;
```

Comment:
supposedly returns `string` but we have type checking turned off for this file :)

You could turn it on, fix any errors in the new code, then turn it back off.